docs/USAGE.md

```diff
@@ -34,7 +34,7 @@ Please note that since downloading will take time, we recommend validating repea
 
 1. [Fork this repository](/docs/REPRODUCE_ERRORS.md#fork-this-repository-detailed-instructions)
```

Comment:
If this is the same as the "Reproduce errors" steps I would remove this section and just point to that file. If they are the same, though, this seems like a much more readable version that's much shorter - I'd prefer this version.

Reply:
It is actually slightly different: explanations in `USAGE.md`  are addressed to someone who would like to run the validator via GitHub actions while explanations in `REPRODUCE_ERRORS.md` are addressed to someone who would like to report a bug.
The difference resides in the fact that to simply run the validator: creating a fork is preferred to avoid noise in our repo; and creating a branch is preferred to report a bug. Updating this documentation to refer `REPRODUCE_ERRORS.md` from `USAGE.md`  in case of bug.